frontend/apps/reader/modules/readeruserhyph.lua

```diff
@@ -120,16 +125,22 @@ function ReaderUserHyph:updateDictionary(word, hyphenation)
         return
     end
 
+    if word then
+        word = Utf8Proc.normalize_NFC(word)
+    end
+
     local word_lower = Utf8Proc.lowercase(word)
```

Comment:
Won't the 2nd one failed if no word ? If not, is it different for the firsts one ?
May be shorten the first one like you did for          `line = line and Utf8Proc.normalize_NFC(line)`

Reply:
The check is not necessary.